server/src/main/java/org/apache/kafka/network/RequestConvertToJson.java

```diff
@@ -465,6 +469,8 @@ public static JsonNode request(AbstractRequest request) {
                 return DescribeShareGroupOffsetsRequestDataJsonConverter.write(((DescribeShareGroupOffsetsRequest) request).data(), request.version());
             case ALTER_SHARE_GROUP_OFFSETS:
                 return AlterShareGroupOffsetsRequestDataJsonConverter.write(((AlterShareGroupOffsetsRequest) request).data(), request.version());
+            case DELETE_SHARE_GROUP_OFFSETS:
```

Comment:
The cases in this switch statement are intended to be in alphabetical order. Please can you move `ALTER_SHARE_GROUP_OFFSETS` and `DELETE_SHARE_GROUP_OFFSETS` in to their correct places in the order. Thanks.